ui/app/templates/components/auth-info.hbs

```diff
@@ -34,6 +34,16 @@
                 Revoke token
               {{/confirm-action}}
             </li>
+            <li class="action">
+              {{#copy-button
```

Comment:
Can we do angle-bracket syntax here too like the `<ICon>` change below?

Reply:
oh yeah!